sdk/keyvault/azure-security-keyvault-secrets/inc/ApiViewSettings.json

```
@@ -0,0 +1,17 @@
+{
+  "sourceFilesToProcess": null,
+  "additionalIncludeDirectories": [
+    "../../../core/azure-core/inc",
+    "../../azure-security-keyvault-shared/inc"
+  ],
+  "sourceFilesToSkip": [
+  ],
+  "additionalCompilerSwitches": [],
+  "allowInternal": false,
+  "includeDetail": false,
+  "includePrivate": false,
+  "filterNamespace": "Azure::Security::KeyVault::Secrets",
+  "reviewName": "KeyVault Secrets Review",
```

Consistency issue, here/elsewhere
```suggestion
  "reviewName": "KeyVault Secrets API Review",
```

I see `Azure Storage Common API Review` below and `Storage Queues Review`. Let's consolidate to one form:
`<package name/service name> API Review`